coalib/bears/Bear.py

```diff
@@ -47,6 +47,12 @@ class Bear(Printer, LogPrinter):
     >>> class SomeBear(Bear):
     ...     LANGUAGES = "Java"
 
+    To see what ``pip`` package requirements the bear has, give the bear a
+    value for ``REQUIREMENTS``, which will be a tuple of strings:
+
+    >>> class SomeBear(Bear):
+    ...     REQUIREMENTS = ('yamllint~=1.0.1',)
```

Comment:
how will coala enforce this requirement? Should we show it at the detailed bear information?